com.unity.render-pipelines.universal/Documentation~/urp-optimization.md

```diff
@@ -0,0 +1,7 @@
+# Optimization
```

Comment:
Should the section name be Optimization and Debugging? 
 It's weird to me it's just called optimization and there's no optimization material referred in this page yet.

Reply:
The Rendering Debugger window has more features for optimization, rather than specifically for debugging, but I can add "Debugging" to the title of course.